docker-compose.yml

```diff
@@ -90,6 +92,40 @@ services:
       - 3000:3000
     restart: unless-stopped
 
+  lnd:
+    container_name: lnd
+    image: lightninglabs/lnd:v0.15.5-beta
+    depends_on:
+      - bitcoin
+    command:
+      - --configfile=/var/lib/lnd/lnd.conf
```

Comment:
Can we pass in the related conf via arg so that we don't have to deal with local filestorage?

Reply:
It should be possible, but as you've pointed out below, we would still need to deal with local file storage.